### ISSUE TITLE
Upgrade my project and my external third library from Angular 11 to 12 fails because of a dependency to "ngx-toastr" in my library

### ISSUE THREAD

**Issue by JTHOUMEL** [scttcper/ngx-toastr#998]
I have two Angular projects : 
- the main Angular project
- an external third library

The external third library has a dependency to the `ngx-toastr` library.

With Angular 11 (`"@angular/core": "~11.2.14"`, `"ngx-toastr": "^13.1.0"`), everything is OK.

After upgrading to Angular 12 (`"@angular/core": "~12.2.17"`, `"ngx-toastr": "^14.3.0"`), I have an error after launching the `ng serve` command:

```
C:\Users\my-path\node_modules\@angular\compiler-cli\ngcc\src\entry_point_finder\targeted_entry_point_finder.js:40
                throw new Error("The target entry-point \"" + invalidTarget.entryPoint.name + "\" has missing dependencies:\n" +
                ^

Error: The target entry-point "my-external-third-library" has missing dependencies:
 - ngx-toastr/public_api
 - ngx-filesaver/index
```

(note the `ngx-filesaver` library is also causing a similar issue)

Any idea of what's wrong ?

See my post on [StackOverflow](https://stackoverflow.com/questions/76389292/upgrade-my-project-and-my-external-third-library-from-angular-11-to-12-fails-bec) for a complete description of the issue.

**Comment by scttcper** [comment]
https://github.com/scttcper/ngx-toastr#dependencies

**Comment by JTHOUMEL** [comment]
I don't understand.

According to your link, for Angular "12.x" the latest ngx-toastr version available is "14.3.0". It is the one I tried.
